js/ball.js

```diff
@@ -105,13 +116,15 @@ export default class Ball {
         }
       }
     }
+
+    //*** this is temporary to be replaced by better algortithm -end- */
   }
 
   collide(ballB) {
-    if (this.state === "healthy" && ballB.state === "sick") {
+    if (this.state === "healthy" && ballB.state === "infected") {
```

Comment:
sick || infected

Reply:
to i tak jest do wymiany całkiem w osobnym zadaniu  https://github.com/qunabu/covid-simulation/issues/3

Reply:
prawda, ale na tą chwilę tamtego nie ma, a będąc chorym na bank zarażasz bardziej niż będąc zainfekowanym :p

Reply:
good point

Reply:
```suggestion
    if (this.state === "healthy" && (ballB.state === "infected" || ballB.state === "sick")) {
```